internal/services/mssql/mssql_elasticpool_resource_test.go

```diff
@@ -268,6 +268,42 @@ func TestAccMsSqlElasticPool_hyperScale(t *testing.T) {
 	})
 }
 
+func TestAccMsSqlElasticPool_vCoreToStandardDTU(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_mssql_elasticpool", "test")
+	r := MsSqlElasticPoolResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.no_licenseType(data),
```

Comment:
We don't use underscores in the test config names, can you please remove it and use camelCase like we do everywhere else in the provider 
```suggestion
			Config: r.noLicenseType(data),
```

Reply:
Fixed.